prater.tf

```diff
@@ -0,0 +1,120 @@
+locals {
+  /* Volumes */
+  prater_root_vol_size = 20
+  prater_data_vol_size = 150
```

Comment:
Pyrmont uses less than 30GB and it's been fine for months now. Maybe we can start with something smaller, like 50GB?

Reply:
50 should be fine for the foreseeable future